### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by nlgwcy**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
https://github.com/kmesh-net/kmesh/issues/470

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bpf/include/bpf_common.h [hunk 1]
```diff
@@ -166,7 +166,7 @@ static inline bool conn_from_cni_sim_delete(struct kmesh_context *kmesh_ctx)
 {
     // cni sim connect CONTROL_CMD_IP:930(0x3a2)
     // 0x3a2 is the specific port handled by the cni to disable Kmesh
-    is_control_connect(kmesh_ctx, CONTROL_CMD_IP, DISABLE_KMESH_PORT);
+    return is_control_connect(kmesh_ctx, CONTROL_CMD_IP, DISABLE_KMESH_PORT);
```
Comment (hzxuzhonghu) [inline bpf/include/bpf_common.h:169]: ....

**Review by hzxuzhonghu — COMMENTED** [review]
But how does this fix `record netcookie failed`, which only has to do with `conn_from_cni_sim_add`

**Comment by nlgwcy** [comment]
> But how does this fix `record netcookie failed`, which only has to do with `conn_from_cni_sim_add`

Because of this mistake, bf bytecode is messed up. I'll submit an issue to clear the bpf complie alarms  later.

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**